internal/logging/defaults.go

```diff
@@ -24,7 +25,9 @@ import (
 // datadir is the base directory at which the log is saved
 var datadir string
 
-var datafile string
+var dataFileName string
+
+var dataFilePath string
```

Comment:
I'm not seeing the necessity for this. It just creates more logic for us maintained to have to grok. And since all of our Filename functions are just concatenating variables together caching doesn't to really serve much of a purpose.

Reply:
The issue is that one of those variables is now a timestamp. The change stems from our use of the `FilePath` function in two places one during [uninstall](https://github.com/ActiveState/cli/blob/master/internal/runners/clean/run_lin_mac.go#L58) and the other when we capture [panics](https://github.com/ActiveState/cli/blob/master/internal/runbits/panics/panics.go#L20). Without caching something (the filename or the timestamp) those functions return a file that does not exist, which was surfaced by integration test failures.

Reply:
Ok, then can we just capture the timestamp in `init()` and only use that as a global? Basically we just want the timestamp to be constant.